### PR TITLE
Update: ecmaVersion default to 2021 in demo (fixes #508)

### LINKED ISSUE #508: Demo default ECMAVersion should be 2018
When the demo is loaded for the first time, the ECMAVersion is set to 5, so it produces errors for all of the new syntax. I think that is confusing, as the ECMAVersion setting is hidden behind a click. Let's set the default ECMAVersion to 2018.

### PR DESCRIPTION
**Pull request by aladdin-add**

### CODE REVIEW

File: src/js/demo/components/App.jsx [hunk 1]
```diff
@@ -7,6 +7,7 @@ import Crash from "./Crash";
 import Configuration from "./Configuration";
 import Unicode from "../utils/unicode";
 import linterModule from "../node_modules/eslint/lib/linter/linter";
+import espree from "espree";
```
[the remark this answers is not in the document]
Comment (aladdin-add) [reply src/js/demo/components/App.jsx:10]: for npm/yarn, it is. but yes, it's not always true. :-

the problem is eslint does not have the latestEcmaVersion. or we can consider add it to eslint's public API?

[the remark this answers is not in the document]
Comment (mdjermanovic) [reply src/js/demo/components/App.jsx:10]: Maybe we could just put hard-coded `12` for now, like we're hard-coding the list:

https://github.com/eslint/website/blob/89a9ce478e1098a4adacd67da2af28421b17620a/src/js/demo/components/ParserOptions.jsx#L14-L22

and open an issue on how to improve this. What do you think?

**Review by nzakas — APPROVED** [review]
LGTM

**Review by mdjermanovic — APPROVED** [review]
LGTM, thanks!

I'm not sure if we already have a consensus for this change.

**Comment by nzakas** [comment]
I think we're good to go here.

**Pull request merged**